ci/code_checks.sh

```diff
@@ -47,7 +47,7 @@ import pandas
 
 blocklist = {'bs4', 'gcsfs', 'html5lib', 'http', 'ipython', 'jinja2', 'hypothesis',
              'lxml', 'matplotlib', 'openpyxl', 'py', 'pytest', 's3fs', 'scipy',
-             'tables', 'urllib.request', 'xlrd', 'xlsxwriter', 'xlwt'}
```

Comment:
keeping xlrd here but removing it in the asv?

Reply:
The asvs use `to_excel/read_excel` to roundtrip test `xlwt/xlrd`. Since `xlwt` is removed, the roundtrip is broken and I thought It's not worth adding a separate, large excel file to test xlrd.